ToDoList-frontend/src/app/bucket/bucket.component.ts

```diff
@@ -117,17 +125,46 @@ export class BucketComponent implements OnInit {
         this.bucketTaskPriorities = response;
       },
       (error: any) => {
-        this.toastr.error("Request failed. Check console logs and network tab to identify the issue." + error.name)
+        this.toastr.error("Request failed")
+      }
+    );
+  }
+
+  fetchAssigness() {
+    this.assigneeService.getAssignees(environment.assigneeEndpoint).subscribe(
+      (response: Assignee[]) => {
+        this.assignees = response;
+      },
+      (error: any) => {
+        this.toastr.error("Request failed")
       }
     );
   }
 
+  toggleShowAssignees() {
+    this.showAssignees = !this.showAssignees;
+  }
+
+  selectAssignee(assignee: Assignee) {
+    this.bucketTaskFormGroup.patchValue({ assignee:assignee.name });
+    this.showAssignees = false;
+  }
+
+  findAssigneeById(assignees: Assignee[], assigneeIdKey: number): string {
```

Comment:
Wydaje mi sie ze nie musimy przekazywac assignees w ogole. Możemy po prostu w środku zrobić `this.assignees.find...`. Ta funkcja zawsze bedzie wywolywana zeby znalezc assignee z listy przechowywanej jako pole w komponencie. ;)

Teraz zauwazylem, ze nazwa funkcji jest troche mylaca. Nazywa sie `findAssignee` a my nie zwracamy calego assignee, tylko jego name. Zwracalbym caly obiekt assignee z tej funkcji. Analogicznie w funkcji ponizej.

Do tego assigneeIdKey -> id, assigneeNameKey -> name